pinot-core/src/main/java/org/apache/pinot/core/common/RowBasedBlockValueFetcher.java

```diff
@@ -43,6 +44,15 @@ public Object[] getRow(int docId) {
     return row;
   }
 
+  public RoaringBitmap getColumnNullBitmap(int colId) {
```

Comment:
I don't think the changes in this file is required since we don't directly read `null` into the row. The nullBitmap is not in row format, so we should directly read it from `BlockValSet` on the caller side

Reply:
left from the early iterations. good catch.